encoder/adderencoding.h

```diff
@@ -14,6 +14,7 @@
 namespace PBLib
 {
 int ld64(const uint64_t x);
+int ld64universal(const uint64_t x);
```

Comment:
Isn't this method just an implementation detail of `ld64`? If it is, it should be locally-linked in the implementation file.